packages/documentation/src/stories/components/stepper/stepper.docs.mdx

```diff
@@ -0,0 +1,53 @@
+import { Meta, Source } from '@storybook/blocks';
+import { PostTabs, PostTabHeader, PostTabPanel } from '@swisspost/design-system-components-react';
+import StylesPackageImport from '../../../shared/styles-package-import.mdx';
+import PostComponentDemoLink from '../../../shared/post-component-demo-link.mdx';
+import NgbComponentImport from '../../../shared/nb-bootstrap/ngb-component-import.mdx';
+import * as stepperStories from './stepper.stories';
+import stepperTemplate from './stepper-template.sample.html?raw';
+import stepperComponent from './stepper-component.sample?raw';
+
+<Meta of={stepperStories} />
+
+<div className="d-flex align-items-center justify-content-between">
+  # Stepper
+
+  <PostComponentDemoLink component="stepper" />
+</div>
+
+<p className="lead">Conveys progress through numbered steps.</p>
+
+<div className="alert alert-info mb-bigger-big">
+  <p className="alert-heading">The stepper uses ng-bootstrap's progressbar component.</p>
+</div>
+
+<ul>
+  <li>
+    <a href="#component-import" target="_self">Component Import</a>
+  </li>
+  <li>
+    <a href="#style-imports" target="_self">Style Imports</a>
+  </li>
+  <li>
+    <a href="#example" target="_self">Example</a>
+  </li>
+</ul>
```

Comment:
These links are not working properly:
![image](https://github.com/swisspost/design-system/assets/9716662/2f89a033-22f9-4566-a96a-10a5e005c930)

Reply:
I have created a ticket to address this problem, because it's happening not only here and its already live 😢
#2575